src/php/inc/theme-setup.php

```diff
@@ -95,3 +95,50 @@ function custom_hide_editor() {
 	}
 }
 add_action( 'admin_init', 'custom_hide_editor' );
+
+/**
+ * Customize the pingbacks and trackbacks option on new posts by default.
+ *
+ * @param mixed  $value   The current option value.
+ * @param string $option The name of the option being filtered.
+ * @param mixed  $default The default value for the option.
+ *
+ * @return mixed The filtered option value. If the provided option is 'default_ping_status',
+ *               this function will set its value to 'closed' (disabling pingbacks and trackbacks),
+ *               and return the updated value. For other options, it returns the original value.
+ */
+function custom_disable_pingbacks_trackbacks_option( $value, $option, $default ) {
+    if ( 'default_ping_status' === $option ) {
+        $value = 'closed';
+    }
+
+    return $value;
+}
+
+add_filter( 'pre_option_default_ping_status', 'custom_disable_pingbacks_trackbacks_option', 10, 3 );
+
+/**
+ * Disable pingbacks and trackbacks on the default Hello world post.
+ */
+function update_hello_world_post_on_init() {
+	$hello_world_post = get_post( 1 );
+
+	if ( $hello_world_post ) {
+		$updated_post_data = array(
+            'ID' => $hello_world_post->ID,
+            'ping_status' => 'closed',
+		);
+
+		$post_id = wp_update_post( $updated_post_data );
+	}
+}
+add_action( 'init', 'update_hello_world_post_on_init' );
+
+/**
+ * Remove Trackbacks Support for the "post" Post Type.
+ */
+function remove_comments_support() {
```

Comment:
Since we're not actually removing comments support, we should rename this to `remove_trackbacks_support`.